WordPress/gutenberg
# Updated unstyledButton hover selector to change fill of svg and path element

## What?
In the sidebar we have list of links to patterns and media files. Inside the button there is text (name of the pattern/name of the file) and svg icon (chevron pointing to right). When hovering over the button fill of svg does not change.

Changed how `:focus` state is handled to be more consistent with another view.

## Why?
Issue(s) were originally mentioned #50758 and in #50757
The goal is to make UX more consistent

## How?
Extended button `:hover` to change fill of `svg` elements inside the button

## Testing Instructions for #50758
1. Open a Post or Page.
2. Open the Inserter (blue background plus icon on the top left)
3. Navigate to "Patters" tab and hover over some button (pattern). See that fill of the svg (icon on the right) changes
4. Navigate to "Media" repeat the step 3.

## Testing Instructions for #50757
1. Open Post or Page
2. Open the Inserter (blue background plus icon on the top left)
3. Navigate to "Patterns" tab and start tabbing the tab to move focus around
4. Navigate to "Media" and start hitting tab

### Testing Instructions for Keyboard


## Screenshots or screencast
